builtins/en/facets/instructions/team-leader-finding-contract-fix.md

```
@@ -1,12 +1,12 @@
-Decompose the Finding Contract's actionable open findings into non-conflicting repair parts and make the final decision for the fix step. The parent Team Leader must not use tools; decide from the engine-provided Finding Contract summary, part claims, and compact index.
+Decompose the Finding Contract's actionable open findings into repair parts and make the final decision for the fix step. The parent Team Leader must not use tools; decide from the engine-provided Finding Contract summary, part claims, and compact index.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)